### ISSUE TITLE
Problem provisioning (wget issue)

### ISSUE THREAD

**Issue by sehz** [hashicorp/learn-terraform-provision-eks-cluster#14]
I am getting following error which something to do with ```wget```.

```
ks.aws_launch_configuration.workers[1]: Still creating... [10s elapsed]
module.eks.aws_launch_configuration.workers[0]: Creation complete after 10s [id=training-eks-W5YjPdfh-worker-group-120200724013458477900000010]
module.eks.aws_launch_configuration.workers[1]: Creation complete after 10s [id=training-eks-W5YjPdfh-worker-group-220200724013458484400000011]
module.eks.random_pet.workers[0]: Creating...
module.eks.random_pet.workers[1]: Creating...
module.eks.random_pet.workers[0]: Creation complete after 0s [id=capable-snipe]
module.eks.random_pet.workers[1]: Creation complete after 0s [id=deep-wombat]
module.eks.aws_autoscaling_group.workers[0]: Creating...
module.eks.aws_autoscaling_group.workers[1]: Creating...
module.eks.null_resource.wait_for_cluster[0] (local-exec): /bin/sh: wget: command not found
module.eks.null_resource.wait_for_cluster[0]: Still creating... [20s elapsed]
module.eks.null_resource.wait_for_cluster[0] (local-exec): /bin/sh: wget: command not found
module.eks.aws_autoscaling_group.workers[0]: Still creating... [10s elapsed]
module.eks.aws_autoscaling_group.workers[1]: Still creating... [10s elapsed]
```

**Comment by MEOWMEOW114** [comment]
I faced the same issue, 
[and found this is the workaround](https://github.com/terraform-aws-modules/terraform-aws-eks/issues/757)

**Comment by im2nguyen** [comment]
Thanks @MEOWMEOW114 

To address this issue, add the following to your `module "eks" {}` in `eks-cluster.tf`.

```diff
module "eks" {
   ...
+  wait_for_cluster_cmd  = "until curl -k -s $ENDPOINT/healthz >/dev/null; do sleep 4; done"
}
```

**Comment by im2nguyen** [comment]
@tanakamapondera , what OS and Terraform version are you using? It seems like your system doesn't have bash.

This https://github.com/terraform-aws-modules/terraform-aws-eks/issues/925#issuecomment-648057547 comment includes a workaround for Window machines. Please let me know if that works!

**Comment by HormCodes** [comment]
On OSX: Installing `wget` using Homebrew can also fix the issue: `brew install wget`.

**Comment by alanszlosek** [comment]
This should now be fixed by #65. Please let us know if you are still having issues.
